prisma/data.ts

```
@@ -5,149 +5,79 @@ export const categories = [
         description: 'Stories featuring animals as main characters.',
     },
     {
-        name: 'Fairytales',
-        image: 'https://images.unsplash.com/photo-1502082553048-f009c37129b9',
-        description: 'Classic fairy tales with magical elements.',
-    },
-    {
-        name: 'Fables and Folktales',
-        image: 'https://images.unsplash.com/photo-1464983953574-0892a716854b',
-        description: 'Stories with morals and lessons from folklore.',
+        name: 'Fantasy & Magic',
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)